src/paper/tasks.py

```diff
@@ -1269,45 +1308,97 @@ def pull_openalex_author_works(user_id, openalex_id):
     return True
 
 
-# Temporarily disabled as it was causing a lot of slowness in production
-# @periodic_task(
-#     # run at 4:30 PM UTC (8:30 AM PST)
-#     run_every=crontab(minute=30, hour=16), priority=3, queue=QUEUE_PULL_PAPERS
-# )
-def pull_new_openalex_works(page=0, retry=0):
+@periodic_task(
+    # run at 4:30 PM UTC (8:30 AM PST)
+    run_every=crontab(minute=30, hour=16), priority=3, queue=QUEUE_PULL_PAPERS
+)
+def pull_new_openalex_works(start_index=0, retry=0, paper_fetch_log_id=None):
+    """
+    Pull new works (papers) from OpenAlex.
+    
+    This looks complicated because we're trying to handle retries and logging.
+    But simply:
+    1. Get new works from OpenAlex in batches
+    2. Kick-off a task to create/update papers for each work
+    3. If we hit an error, retry the job from where we left off
+    4. Log the results
+    """
     if not PRODUCTION:
         return
 
     from paper.models import PaperFetchLog
 
-    sentry.log_info("Starting New OpenAlex pull")
+    fetch_since_date = datetime.now() - timedelta(days=1)
```

Comment:
What does `fetch_since_date` represent? I don't think I fully understand what it means. Is it meant to represent the last time we tried fetching aka `last_fetch_date` ?

Reply:
I think I got it. It represent the last date we fetched papers from. Kinda minor, but can we change to `fetch_from_date` or `date_to_fetch_from` since due to the nature of SQL, it is a bit clearer what is happening.

Reply:
Updated!